protocol/0062-SPAM-spam_protection.md

```diff
@@ -85,19 +85,28 @@ Further, each party is allowed to submit up to `n` transactions per epoch where
 
 ### Transaction Spam
 
+Before any order or liquidity commitment is accepted for a perpetual futures or expiring futures check that the party has `margin + general > 0` with `margin` being the balance in the margin account for the relevant market and `general` the balance in the general account for the relevant asset. Orders from parties that don't meet this criteria are rejected. This is to be done after the PoW check.
```

Comment:
we need to account for order margin I'd imagine too

Reply:
Yes, please have a look again after the recent edit.